src/Domain/Syntax/Wildcard.php

```diff
@@ -12,25 +12,33 @@ class Wildcard implements SyntaxInterface
 
     private float $boost;
 
+    private bool $caseInsensitive = false;
+
+    private ?string $rewrite = null;
+
     public function __construct(
         string $field,
         string $value,
-        float $boost = 1.0
+        float $boost = 1.0,
+        bool $caseInsensitive = false,
+        string $rewrite = null,
```

Comment:
`null` is not a valid value according to https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-term-rewrite.html

Two options: 
1. make it optional and not include it in the query if it is null.
2. set it to the default value.

I have no preference I think.

Reply:
Fixed!